config/environments/development.rb

```diff
@@ -6,8 +6,9 @@
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
 
-  config.hosts << 'lvh.me'
-  config.hosts << /.*\.lvh.me/
+  # config.hosts << 'lvh.me'
+  # config.hosts << /.*\.lvh.me/
+  config.hosts.clear
```

Comment:
We should put a comment just stating the impetus for this i think? For the future